rl_loop.py

```diff
@@ -140,18 +145,35 @@ def gather():
                 output_directory=TRAINING_CHUNK_DIR)
 
 
-def train():
+def train(logdir=None, load_dir=MODELS_DIR, save_dir=MODELS_DIR):
```

Comment:
Is logdir used at all? Also, in what situations would you override load_dir / save_dir?

Reply:
hmm.  Logdir was definitely important, i'll revert wherever that got lost.

load_dir/save_dir is pretty useful when you want to try to train new models but not put them in the same bucket that's 'live' for selfplay.

Reply:
oh, i guess the estimator uses the working_dir to write out the tensorboard logs.  I guess we can blitz it.